packages/cli/src/utils/deploy-v2.ts

```diff
@@ -105,7 +105,11 @@ export async function deploy(mudConfig: MUDConfig, deployConfig: DeployConfig):
   const contractPromises: Record<string, Promise<string>> = { ...worldPromise, ...systemPromises, ...modulePromises };
 
   // Create World contract instance from deployed address
-  const WorldContract = new ethers.Contract(await contractPromises.World, IBaseWorldData.abi, signer) as IBaseWorld;
+  const WorldContract = new ethers.Contract(
+    await contractPromises.World,
+    IBaseWorldData.abi,
+    signer
+  ) as any as IBaseWorld;
```

Comment:
why do we need to cast to `any` first here?

Reply:
might have been related to me chasing down bugs with circular dependencies + ethers version misalignment, lemme revert and see if this still works

Reply:
was able to revert this